### PR TITLE
improve contrast of top

### PR DESCRIPTION
**Pull request by rohini-ranjanR**
### Description
Change color to improve contrast of the top.

### Related issues and pull requests
issue #2054

### CODE REVIEW

File: live-examples/css-examples/positioned-layout/offsets.css [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 #example-element {
     background-color: #ff7100;
-    border: 3px solid red;
+    border: 3px solid #00ff58;
```
Comment (NiedziolkaMichal) [inline live-examples/css-examples/positioned-layout/offsets.css]: This change doesn't fix the contrast issue. Also, the green border and the orange background don't look good.

File: live-examples/css-examples/text-decoration/text-emphasis.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
     </div>
 
     <div class="example-choice">
-        <pre><code class="language-css">text-emphasis: 'x';</code></pre>
+        <pre><code class="language-css">text-emphasis: <code style="color: #d8ff00">'x'</code>;</code></pre>
```
Comment (NiedziolkaMichal) [inline live-examples/css-examples/text-decoration/text-emphasis.html]: This shouldn't be changed. The colors of CSS declarations are controlled by BOB.

**Review by NiedziolkaMichal — CHANGES_REQUESTED** [review]
Changes to `transform-origin` look good. I have added comments about changes to the other files.

**Comment by NiedziolkaMichal** [comment]
It looks like you have 4 pull requests, all of which have the same proposed changes. When you wish to make a pull request with changes to a single interactive example or group of related examples, please try to create it on a separate branch from other PRs. After that when you wish to make another pull request, make a new branch from the `main` branch of `interactive-examples` repository, instead of from a branch you have created for a separate pull request. This way, we can review each set of changes independently.

File: live-examples/css-examples/positioned-layout/offsets.css (hunk 1 above)
Comment (rohini-ranjanR) [reply live-examples/css-examples/positioned-layout/offsets.css]: Thanks, @NiedziolkaMichal. I make the changes

**Comment by rohini-ranjanR** [comment]
> It looks like you have 4 pull requests, all of which have the same proposed changes. When you wish to make a pull request with changes to a single interactive example or group of related examples, please try to create it on a separate branch from other PRs. After that when you wish to make another pull request, make a new branch from the `main` branch of `interactive-examples` repository, instead of from a branch you have created for a separate pull request. This way, we can review each set of changes independently

I'll keep that in mind next time. Thanks, @NiedziolkaMichal

**Review by NiedziolkaMichal — APPROVED** [review]
Great job! Thank you very much

**Comment by rohini-ranjanR** [comment]
Thank you @NiedziolkaMichal

**Pull request merged**